spec/controllers/step_nav_controller_spec.rb

```diff
@@ -1,6 +1,4 @@
-require "test_helper"
-
-describe StepNavController do
+RSpec.describe StepNavController do
   it "returns a 404 when the page doesn't exist" do
     slug = SecureRandom.hex
     stub_content_store_does_not_have_item("/#{slug}")
```

Comment:
looks like assert_response isn't picked up by the tool

assert_response 404 --> `expect(response).to have_http_status(:not_found)`

assert_response 403 --> `expect(response).to have_http_status(:forbidden)`